trufflehog.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: trufflehog
+  version: 3.88.5
+  epoch: 0
+  description: Find, verify, and analyze leaked credentials
+  dependencies:
+    runtime:
+      - bash
+      - binutils
+      - ca-certificates
+      - cpio
+      - git
+      - openssh-client
```

Comment:
Once https://github.com/wolfi-dev/os/pull/42235 is merged, add rpm2cpio as runtime dependency here.